definitions/tools/biscuit_align.cwl

```diff
@@ -3,22 +3,32 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Biscuit: align"
-baseCommand: ["/usr/bin/biscuit","align"]
+baseCommand: ["/bin/bash","biscuit_align.sh"]
 requirements:
     - class: ShellCommandRequirement
     - class: ResourceRequirement
       ramMin: 32000
       coresMin: 12
     - class: DockerRequirement
-      dockerPull: "mgibio/bisulfite:v1.3"
+      dockerPull: "mgibio/biscuit:0.3.8"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'biscuit_align.sh'
+        entry: |
+            set -eo pipefail
+
+            cores=$1
+            outdir=$2
+            read_group_id="$3"
+            reference_index=$4
+            fastq1=$5
+            fastq2=$6
+
+            /usr/bin/biscuit align -t $cores -M -R "$read_group_id" $reference_index $fastq1 $fastq2 | /usr/bin/sambamba view -S -f bam -l 0 /dev/stdin | /usr/bin/sambamba sort -t $cores -m 8G -o $outdir/aligned.bam /dev/stdin
+
 arguments: [
-    { valueFrom: "-t", position: -10 },
     { valueFrom: $(runtime.cores), position: -9 },
-    { valueFrom: "-M", position: -8 },
-    { shellQuote: false, valueFrom: "|" },
-    "/usr/bin/sambamba", "view", "-S", "-f", "bam", "-l", "0", "/dev/stdin",
-    { shellQuote: false, valueFrom: "|" },
-    "/usr/bin/sambamba", "sort", "-t", $(runtime.cores), "-m", "8G", "-o", "$(runtime.outdir)/aligned.bam", "/dev/stdin"
+    { valueFrom: $(runtime.outdir), position: -8 },
```

Comment:
At this point since we're adding positions to everything, it might be nice to use position numbers that correspond to the bash variable numbers.  Not a requirement, of course!